src/alert-manager/src/alert-handler/routes/actions.js

```diff
@@ -24,12 +24,12 @@ const router = express.Router();
 
 // email
 router
-  .route('/alert-handler/send-email-to-admin')
+  .route('/alert-handler/send-email-to-admin/:template*?')
```

Comment:
Change API to `/alert-handler/send-email-to-admin?template=template-name`?

Reply:
> Change API to `/alert-handler/send-email-to-admin?template=template-name`?

Changed, thanks.